src/err/mod.rs

```diff
@@ -260,6 +278,18 @@ impl PyErr {
         self.normalized(py).ptype(py)
     }
 
+    /// Deprecated form of [`PyErr::value_bound`].
+    #[cfg_attr(
+        not(feature = "gil-refs"),
+        deprecated(
+            since = "0.21.0",
+            note = "`PyErr::value` will be replaced by `PyErr::value_bound` in a future PyO3 version"
+        )
+    )]
+    pub fn value<'py>(&'py self, py: Python<'py>) -> &'py PyBaseException {
+        self.value_bound(py).clone().into_gil_ref()
```

Comment:
One _final_ place where we can avoid a clone 😄 

```suggestion
        self.value_bound(py).as_gil_ref()
```

(This works because the constraint `&'py self` means that this `self.value_bound(py)` here is `&'py Bound<'py, PyAny>`, which is the right form for `.as_gil_ref()`.)